doc/cascadia/profiles.schema.json

```diff
@@ -564,6 +564,10 @@
           ],
           "type": "string"
         },
+        "retroTerminalEffect": {
+          "description": "When set to true, enable retro terminal effects.",
+          "type": "boolean"
+        }
```

Comment:
Should be a comma here.

Reply:
Looks like this was already fixed 
https://github.com/microsoft/terminal/blob/82f302b7142e116c52f5c143761febb76b34a9b1/doc/cascadia/profiles.schema.json#L535-L538